x-pack/plugins/security_solution/public/detections/components/rules/related_integrations/integrations_popover/index.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiPopover,
+  EuiBadge,
+  EuiPopoverTitle,
+  EuiFlexGroup,
+  EuiText,
+  EuiSpacer,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { IntegrationDescription } from '../integrations_description';
+import { getInstalledRelatedIntegrations } from '../utils';
+import { useInstalledIntegrations } from '../use_installed_integrations';
+import type { RelatedIntegrationArray } from '../../../../../../common/detection_engine/schemas/common';
+
+import * as i18n from '../translations';
+
+export interface IntegrationsPopoverProps {
+  integrations: RelatedIntegrationArray;
+}
+
+const IntegrationsPopoverWrapper = styled(EuiFlexGroup)`
+  width: 100%;
+`;
+
+const PopoverTitleWrapper = styled(EuiPopoverTitle)`
+  max-width: 390px;
+`;
+
+const PopoverContentWrapper = styled('div')`
+  max-height: 400px;
+  max-width: 390px;
+  overflow: auto;
+  line-height: ${({ theme }) => theme.eui.euiLineHeight};
+`;
+
+const IntegrationListItem = styled('li')`
+  list-style-type: disc;
+  margin-left: 25px;
+  margin-bottom: 5px;
+`;
+
+/**
+ * Component to render installed and available integrations
+ * @param integrations - array of integrations to display
+ */
+const IntegrationsPopoverComponent = ({ integrations }: IntegrationsPopoverProps) => {
+  const [isPopoverOpen, setPopoverOpen] = useState(false);
+  const { data: allInstalledIntegrations } = useInstalledIntegrations({ packages: [] });
+
+  const integrationDetails = getInstalledRelatedIntegrations(
+    integrations,
+    allInstalledIntegrations
+  );
+
+  const totalRelatedIntegrationsInstalled = integrationDetails.filter((i) => i.is_enabled).length;
```

Comment:
Nit: `totalRelatedIntegrationsEnabled`

Reply:
Ended up being resolved in https://github.com/elastic/kibana/pull/134299